includes/templates/class-amp-post-template.php

```diff
@@ -342,8 +342,14 @@ private function build_post_comments_data() {
 	 * Build post content.
 	 */
 	private function build_post_content() {
-		/** This filter is documented in wp-includes/post-template.php */
-		$content = apply_filters( 'the_content', get_the_content( null, false, $this->post ) );
```

Comment:
Note that this did not need the `post_password_required()` check before because `get_the_content()` has that logic inside of it.